{
    "Computer":  "SERVER2019",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1759409010800)\/",
                       "Source":  "LHWinService",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "  There may be a configuration problem: please check the logs.",
                       "Id":  100,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  252057
                   },
                   {
                       "TimeCreated":  "\/Date(1759392715808)\/",
                       "Source":  "Microsoft-Windows-WindowsUpdateClient",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "Installation Failure: Windows failed to install the following update with error 0x80073701: 2025-09 Cumulative Update for Windows Server 2019 (1809) for x64-based Systems (KB5065428).",
                       "Id":  20,
                       "ErrorCode":  "0x80073701",
                       "UpdateTitle":  "2025-09 Cumulative Update for Windows Server 2019 (1809) for x64-based Systems (KB5065428)",
                       "EventRecordID":  2883740
                   },
                   {
                       "TimeCreated":  "\/Date(1759378141360)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Unexpected error querying for the IVssWriterCallback interface.  hr = 0x80070005, Access is denied.\r",
                       "Id":  8194,
                       "ErrorCode":  "0x80070005",
                       "UpdateTitle":  null,
                       "EventRecordID":  251751
                   },
                   {
                       "TimeCreated":  "\/Date(1759334381704)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "A timeout was reached (30000 milliseconds) while waiting for the NexHealth Synchronizer service to connect.",
                       "Id":  7009,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  2881575
                   },
                   {
                       "TimeCreated":  "\/Date(1759334381704)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The NexHealth Synchronizer service failed to start due to the following error: \r",
                       "Id":  7000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  2881576
                   },
                   {
                       "TimeCreated":  "\/Date(1759334350908)\/",
                       "Source":  "NexHealthSync",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "SetServiceStatus failed setting STOPPED status Error 6 - The handle is invalid.",
                       "Id":  242,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  251375
                   },
                   {
                       "TimeCreated":  "\/Date(1759334350908)\/",
                       "Source":  "NexHealthSync",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "The instance\u0027s SvcRun() method failed \r",
                       "Id":  3,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  251374
                   },
                   {
                       "TimeCreated":  "\/Date(1759334350718)\/",
                       "Source":  "Microsoft-Windows-RestartManager",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Application or service \u0027NexHealth Synchronizer\u0027 could not be shut down.",
                       "Id":  10006,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  251376
                   }
               ],
    "LastPoll":  "10/02 06:37"
}